src/platform/forms-system/src/js/containers/FormApp.jsx

```diff
@@ -36,7 +36,9 @@ class FormApp extends React.Component {
     const trimmedPathname = currentLocation.pathname.replace(/\/$/, '');
     const lastPathComponent = currentLocation.pathname.split('/').pop();
     const isIntroductionPage = trimmedPathname.endsWith('introduction');
-    const isNonFormPage = this.nonFormPages.includes(lastPathComponent);
+    const isPageNotFound = hasPageNotFound(children);
+    const isNonFormPage =
```

Comment:
Note: isNonFormPage removes a lot of the elements we don't need for the PageNotFound.